re-search.c

```diff
@@ -49,8 +49,12 @@
 #ifndef PROMPT
 #define PROMPT(buffer, saved, direction, index, result) \
 	do { \
-		/* print the first part of the prompt */ \
-		fprintf(stderr, "%s<%s search> %s%s", saved, direction, CYAN, buffer); \
+    /* print the subsearch */ \
```

Comment:
Invalid indentation

Reply:
Damn, I always don't notice that, since I usually use spaces for indentation.
It is fixed now (rewrote history).